frontend/routes/signUp/signUp.component.tsx

```diff
@@ -129,16 +125,12 @@ export class SignUp extends React.PureComponent<IProps, IState> {
 	public handleSubmit = (values, form) => {
 		const data = omit(values, 'username', 'emailConfirm', 'passwordConfirm', 'termsAgreed');
 		this.props.onRegister(values.username, data);
-		this.resetForm(form);
-	}
-
-	public resetForm = (form) => {
-		form.resetForm();
+		values.password = '';
```

Comment:
@carmenfan Sorry but you can't do that in this way. It probably works, but only because of object's mutability.
Instead, you can use `form.setFieldValue: (field: string, value: any, shouldValidate?: boolean)` or `form.resetForm: (nextValues?: Values)` to achieve effect, that you want. ;)

Let me know if these solutions won't work.

Reply:
Thanks! I tried for a while to find alternatives to do it, looked at Formik's Documentation and couldn't find their API for forms so I kind of gave up and see what you'd say 😆 

This is now changed.